content/logs/aws.md

```diff
@@ -52,7 +52,10 @@ Push your AWS log information to Datadog using Lambda functions that respond to
 
 Your Lambda function is now ready to send logs to the Datadog platform. 
 
-There are two ways to configure the triggers that will cause the Lambda to execute and send logs to Datadog. We can [automatically](#automatically-set-up-triggers) manage them for you if you grant us a set of [permissions](#permissions), or you can [manually](#manually-set-up-triggers) set up each trigger yourself in the AWS console.
+There are two ways to configure the triggers that will cause the Lambda to execute and send logs to Datadog. 
+
+* [automatically](#automatically-set-up-triggers): we manage them for you if you grant us a set of [permissions](#permissions)
```

Comment:
r/we manage them for you if you grant us a set of [permissions](#permissions)/
With [the right set of permissions](#permissions), Datadog manages them for you.